server/notification-providers/ntfy.js

```diff
@@ -13,6 +13,7 @@ class Ntfy extends NotificationProvider {
                 "message": msg,
                 "priority": notification.ntfyPriority || 4,
                 "title": "Uptime-Kuma",
+                "icon": notification.ntfyIcon || "",
```

Comment:
> X-Icon | Icon | URL to use as notification icon

The api doc do not say anything about empty string, I think it is not a proper handling.

Reply:
The docs aren't really clear there. In current ntfy releases this works, but I changed the notification provider to only add the Icon header if there actually is an icon.